packages/lib/src/io-command.ts

```diff
@@ -128,6 +128,15 @@ function writeOutputPrivate<T>(data: T, outputOptions: OutputOptions,
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface Inputting<I> extends Loggable {}
+/**
+ * The "Inputting" mixin is used to add the ability of accepting complex
+ * input in the form of JSON or YAML to a command.
```

Comment:
May want to specify as a filepath/name.

Reply:
Added the text "(from stdin or a file)".